cmd/main.go

```diff
@@ -77,6 +78,7 @@ var channels = plugins.Channels{
 
 var report = reporting.Init()
 var secretsChan = make(chan reporting.Secret)
+var errorChan = make(chan error, 1)
```

Comment:
I think I don't see where you are using it.

Reply:
Removed it